pam.d/system-login

```diff
@@ -1,11 +1,9 @@
-auth		required	pam_tally2.so onerr=succeed
 auth		required	pam_nologin.so
 auth		include		system-auth
  				
 account		required	pam_access.so
 account		required	pam_nologin.so
 account		include		system-auth
-account		required	pam_tally2.so onerr=succeed
```

Comment:
Should we move to pam_faillock or just do what upstream did and drop tally without using faillock?

Reply:
I am not sure.
Anyway the upstream [pambase](https://github.com/gentoo/pambase/tree/3f36e2c3de28b3cde25a27d05e49d354e098c368/templates) seems to have removed pam_tally2.so.

Reply:
`pam_faillock` should be used instead and it's in the upstream https://github.com/gentoo/pambase/blob/3f36e2c3de28b3cde25a27d05e49d354e098c368/templates/system-auth.tpl file but not in our https://github.com/kinvolk/baselayout/blob/flatcar-master/pam.d/system-auth file. Maybe it makes sense to update the system-auth file

Reply:
But they are very different and I don't know this area well… maybe it's something for a follow-up PR.

Reply:
ok, let's address the `pam_faillock` issue later in a separate PR.